etc/skydive.yml.default

```diff
@@ -75,17 +75,6 @@ analyzer:
     # Max number of flows in write buffer (after which all flows accumulated are dropped)
     # max_flow_buffer_size: 100000
 
-    # If a limit is specified, when the index reaches it, it is rolled.
```

Comment:
Those parameters still exist, they just need to moved to the right section

Reply:
yes they exist under storage.elastricsearch or storage.newbackend.driver = elasticsearch
but should not exist under analyzer.flow